src/quo2/components/dropdowns/dropdown/properties.cljs

```diff
@@ -0,0 +1,129 @@
+(ns quo2.components.dropdowns.dropdown.properties
+  (:require [quo2.foundations.colors :as colors]))
+
+(def backgrounds #{:photo :blur})
+
+(def sizes
+  {:size-40 {:size             40
+             :icon-size        20
+             :emoji-size       20
+             :emoji-padding    10
+             :padding-vertical 9
+             :padding-left     {:default 16 :icon 12}
+             :padding-right    12
+             :border-radius    12
+             :text-size        :paragraph-1}
+   :size-32 {:size             32
+             :icon-size        20
+             :emoji-size       20
+             :emoji-padding    6
+             :padding-vertical 5
+             :padding-left     {:default 12 :icon 8}
+             :padding-right    8
+             :border-radius    10
+             :text-size        :paragraph-1}
+   :size-24 {:size             24
+             :icon-size        12
+             :emoji-size       12
+             :emoji-padding    6
+             :padding-vertical 3
+             :padding-left     {:default 8 :icon 8}
+             :padding-right    8
+             :border-radius    8
+             :text-size        :paragraph-2}})
+
+(defn- custom-color-type
+  [theme customization-color active?]
+  {:left-icon-color  colors/white-opa-70
+   :right-icon-color {:color   colors/white-opa-20
+                      :color-2 colors/white}
+   :label-color      colors/white
+   :background-color (if active?
+                       (colors/theme-colors
+                        (colors/custom-color customization-color 60)
+                        (colors/custom-color customization-color 50)
+                        theme)
+                       (colors/theme-colors
+                        (colors/custom-color customization-color 50)
+                        (colors/custom-color customization-color 60)
+                        theme))})
+
+(def sizes-to-exclude-blur-in-photo-bg #{:size-40})
+
+(defn- grey-photo
+  [theme active? size]
+  {:left-icon-color    (colors/theme-colors colors/neutral-80-opa-40 colors/white-opa-40 theme)
+   :right-icon-color   {:color   (colors/theme-colors colors/neutral-80-opa-10 colors/white-opa-10 theme)
+                        :color-2 (colors/theme-colors colors/neutral-100 colors/white theme)}
+   :label-color        (colors/theme-colors colors/neutral-100 colors/white theme)
+   :blur-overlay-color (when (and (= theme :dark) (nil? (sizes-to-exclude-blur-in-photo-bg size)))
+                         (if active? colors/neutral-80-opa-50 colors/neutral-80-opa-40))
+   :blur-type          (if (= theme :light) :light :dark)
+   :background-color   (cond
+                         (= theme :light)
+                         (if active? colors/white-opa-50 colors/white-opa-40)
+
+                         (and (= theme :dark) (sizes-to-exclude-blur-in-photo-bg size))
+                         (if active? colors/neutral-80-opa-50 colors/neutral-80-opa-40))})
```

Comment:
From my perspective, if statements shouldn't be nested, instead they should be changed to a `cond`. So I find a little bad using a `cond` with `if`s inside.

I'd suggest to change those `if`s to be part of the `cond`
 (i.e. `(and (= theme :light) active?)`).

@ilmotta this one is interesting, wdyt?

Reply:
@ulisesmac In many occasions, `cond` ends up being harmful from my point of view, because in the attempt to linearize the conditions, it brings duplication. Here, I think @smohamedjavid wanted to be more DRY at the expense of one nested condition. I usually don't bother about this problem, unless there are two or more levels of condition nesting, but here is just one.